css/style.css

```diff
@@ -1,19 +1,253 @@
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main h1 {
+  font-size: 72px;
+  font-weight: 300;
+}
+
+h2 {
+  font-size: 40px;
+  margin-bottom: 4rem;
+  font-weight: 100;
+}
+
+.btn {
+  color: var(--white);
+  background-color: var(--orange-dark);
+  border-radius: 3px;
+  border: none;
+  padding: 10px 24px;
+}
+
+.btn:hover {
+  color: var(--orange-dark);
+  background-color: var(--white);
+}
+
+/* Service section Under Main Image */
+
+.service__section {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 2rem;
+}
+
+.service__thumbnail {
+  max-height: 100px;
+}
+
+.service {
+  text-align: center;
+  padding: 30px 60px;
+}
+
+.heading__underMainImage,
+.service__summary {
+  color: var(--grey-dark);
+  text-align: center;
+}
+
+.heading__underMainImage {
+  margin-top: 5rem;
+  margin-bottom: 1rem;
+}
+
+/* Advert section */
+
+.advert__section {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: pink;
+  margin: 5rem 0;
+}
+
+.advert__board {
+  text-align: center;
+}
+
+.advert__slogan {
+  color: var(--grey-dark);
+  font-size: 34px;
+  font-style: italic;
+  padding: 0 422px;
+}
+
+/* Footer */
+
+.footer__socialMedia {
+  display: flex;
+  justify-content: center;
+}
+.footer__topBorder {
+  border-top: 1px var(--grey-light) solid;
+  width: 80%;
+  font-weight: 100;
+}
+
+.heading__footer {
+  color: var(--grey-dark);
+  text-align: center;
+  font-weight: 100;
+}
+
+.heading__copyright {
+  color: var(--grey-light);
+  text-align: center;
+  font-weight: 100;
+  margin-bottom: 4rem;
+}
+
+.footer__thumbnail {
+  max-height: 15px;
+  width: auto;
+  border: 1px var(--grey-light) solid;
+  border-radius: 50%;
+  padding: 5px;
+  margin: 7px;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+/* Media queries */
+
+@media screen and (max-width: 992px) {
+  .container {
+    flex-direction: column;
+  }
+
+  .service__section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
 
+  .nav__mobile {
+    display: block;
+    padding: 15px;
+  }
 
+  .nav__list {
+    display: none;
+  }
+
+  .header {
+    padding: 0;
+  }
+
+  .header__logo {
+    max-height: 30px;
+    width: 60px;
+  }
+
+  .main h1 {
+    font-size: 50px;
+    font-weight: 300;
+  }
+
+  h2 {
+    font-size: 25px;
+    margin-bottom: 4rem;
+    font-weight: 100;
+  }
+
+  .advert__section {
+    display: block;
+  }
+
+  .advert__slogan {
+    color: var(--grey-dark);
+    font-size: 34px;
+    font-style: italic;
+    padding: 0 28px;
+  }
+
+  img {
+    max-width: 24rem;
+  }
+
+  .advert__section .btn {
```

Comment:
Id more this .advert-section underneath the other advert-section tag. Makes it easier to follow